addons/mrp/tests/common.py

```diff
@@ -57,9 +57,12 @@ def setUpClass(cls):
         user_group_mrp_byproducts = cls.env.ref('mrp.group_mrp_byproducts')
 
         # Update demo products
-        (cls.product_2 | cls.product_3 | cls.product_4 | cls.product_5 | cls.product_6 | cls.product_7_3 | cls.product_8).write({
+        (cls.product_2 | cls.product_3 | cls.product_4 | cls.product_6 | cls.product_8).write({
             'type': 'product',
         })
+        (cls.product_5 | cls.product_7_3).write({
```

Comment:
`cls.product_7_3` is already consumable no ?

Reply:
Yes, actually `product_5` is already consumable too. They were storable before this fix, I wanted to highlight their new type. I removed the lines.